CMakeLists.txt

```diff
@@ -873,6 +873,9 @@ if(WITH_OPTIM)
             add_definitions(-DS390_DFLTCC_INFLATE)
             list(APPEND ZLIB_ARCH_SRCS ${ARCHDIR}/dfltcc_inflate.c)
         endif()
+		if(WITH_DFLTCC_DEFLATE OR WITH_DFLTCC_INFLATE)
```

Comment:
Here and in a few other places there are tabs instead of spaces.

Reply:
will fix